RocketChat/Rocket.Chat
# fix!: "Forgot Password" e-mail is not translated correctly

## Proposed changes (including videos or screenshots)

 - Add migration to update translation key used by the `Forgot_Password_Email` setting.

## Issue(s)

Introduced by #31343 

## Steps to test or reproduce

Start a workspace on 6.5 (or an older version), then update to the latest RC version. Then, check the e-mail received when triggering the "forgot password" flow.
**_Current behavior:_** the e-mail contains the text `Lets_get_you_new_one` (not correctly translated key).
**_Expected behavior:_** the e-mail should contain a translated text (such as "Let's get you a new one!") instead of a key.

## Further comments


[CORE-247](https://rocketchat.atlassian.net/browse/CORE-247)

[CORE-247]: https://rocketchat.atlassian.net/browse/CORE-247?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
